### DISCUSSION TITLE
Sending Multicast

### DISCUSSION THREAD

**Discussion by JaesukLim** [ANLAB-KAIST/KENSv3#243] [Project Q&A 3. Internet Layer (IP/Routing)] [answered]
To send multicast, is it enough for setting dest_port in ip header to 0xe0000005? or do I have to use htonl?
Actually I sent the packet and it seems it doesn't go to other router and return to itself.

**Comment by Pusnow** [comment, marked as answer]
In IP header, there is no `dest_port`. Did you mean `dest_ip`?

For `dest_ip`, it is enough to set `0xe0000005`, and yes, you should use `htonl` if you are using the integer form.

> Actually I sent the packet and it seems it doesn't go to other router and return to itself.

Please, check you set right `src_ip` and see pcap files.
